packages/elements/src/components/ino-datepicker/local.ts

```diff
@@ -1,29 +1,29 @@
-import { German } from "flatpickr/dist/l10n/de"
-import { english as English } from "flatpickr/dist/l10n/default"
-import { CustomLocale } from 'flatpickr/dist/types/locale';
+import { German } from "flatpickr/dist/l10n/de";
+import { english as English } from "flatpickr/dist/l10n/default";
+import { CustomLocale } from "flatpickr/dist/types/locale";
 
 // Supported Languages
-export type DatepickerLocale = 'de'|'en';
+export type DatepickerLocale = "de"|"en";
 
 // Map: Language Key => Flatpickr language
 export const datepickerLocales: Record<DatepickerLocale, CustomLocale> = {
-  'de': German,
-  'en': English
-}
+	"de": German,
```

Comment:
This seems to collide with our prettier rules which formats every `""` to `''`

Reply:
Oh okay I didn't realize that we're using single quotes for some reason - I'll look into it

Reply:
fixed ✅